docs/quickstart/infrastructure.md

```diff
@@ -61,20 +93,25 @@ overview](/docs/visualize/overview-infrastructure.md#single-node-dashboards) doc
 dashboards and charts](/docs/visualize/interact-dashboards-charts.md) to get the most from all of Netdata's real-time
 metrics.
 
-## Create new dashboards
+### Create new dashboards
 
 You can use Netdata Cloud to create new dashboards that match your infrastructure's topology or help you diagnose
 complex issues by aggregating correlated charts from any number of nodes. For example, you could monitor the system CPU
 from every node in your infrastructure on a single dashboard.
 
 ![An example system CPU
 dashboard](https://user-images.githubusercontent.com/1153921/95915568-2db63400-0d5c-11eb-92cc-3c61cb6519dd.png)
-)
 
 Read more about [creating new dashboards](/docs/visualize/create-dashboards.md) for more details about the process and
 additional tips on best leveraging the feature to help you troubleshoot complex performance problems.
 
-## Configure your nodes
+## Set up your nodes
+
+Netdata Cloud's dashboards and visualizations are enriched by the metrics streamed on-demand from your individual nodes.
+You should ensure each node is configured based on its function, and the applications it runs, so that you have full
```

Comment:
This is a bit vague I think. What is the objective of this phrase?

Reply:
I attempted to make this clearer. It's trying to say that there's value in learning how to configure Agents, because that's how you maximize the value you get out of Cloud.

Reply:
You should optimize each Netdata Agent for the particular environment and application that you are using it with. IoT applications might require less frequent collection intervals or a large number of collected metrics could require changes in the configuration of the dbengine....

Reply:
Just saw your changes, you can disregard my suggestion.